adhocracy-plus/assets/scss/components/_blocks.scss

```diff
@@ -286,3 +286,8 @@ $block-padding-md: 7.5 * $padding;
 .block-quote__author {
     font-family: $font-family-serif;
 }
+
+.accordion__body ul {
+    list-style: initial !important;
+    padding: 0 1rem !important;
+}
```

Comment:
This should be in the accordion component file. We actually already set the styling in there to none, did you check both richtext accordions? (wagtail and ckeditor) I assume we did that for a reason so would test to check if reason is due to design breakages.
code line:
https://github.com/liqd/adhocracy-plus/blob/ddd1853a412abb91748046db4837b3ffb9eb6661/adhocracy-plus/assets/scss/components/_accordion.scss#L43

Reply:
@phillimorland thanks for the hint! I removed this reset, because i cannot see why it is there. This way it is working fine for wagtail (faq blocks) and ckeditor. Can't see in code, why it was reset though.